wpilib/wpilib/command/scheduler.py

```diff
@@ -217,52 +217,31 @@ def enable(self):
         """
         self.disabled = False
 
-    def getName(self):
-        return "Scheduler"
-
-    def getType(self):
-        return "Scheduler"
-
-    def initTable(self, subtable):
-        self.commands = []
-        self.ids = []
-        self.toCancel = []
-
-        if subtable is not None:
-            self.namesEntry = subtable.getEntry("Names")
-            self.idsEntry = subtable.getEntry("Ids")
-            self.cancelEntry = subtable.getEntry("Cancel")
-
-            self.namesEntry.setStringArray(self.commands)
-            self.idsEntry.setDoubleArray(self.ids)
-            self.cancelEntry.setDoubleArray(self.toCancel)
-        else:
-            self.namesEntry = None
-            self.idsEntry = None
-            self.cancelEntry = None
+    def initSendable(self, builder):
+        builder.setSmartDashboardType("Scheduler")
+        self.namesEntry = builder.getEntry("Names")
+        self.idsEntry = builder.getEntry("Ids")
+        self.cancelEntry = builder.getEntry("Cancel")
+        builder.setUpdateTable(self.updateTable)
 
     def updateTable(self):
```

Comment:
Mark this as private ­(leading underscore), since this isn't intended to be exposed to users.